.github/workflows/make-test.yml

```diff
@@ -0,0 +1,110 @@
+name: 'Make Test'
+on:
+  push:
+    branches:
+      - maint-3.9
```

Comment:
The containers and the deps therein are specific to 3.9, so this is intentional. The same would work, right now, for master. But that should be a separate PR.